app/api/models/user.py

```diff
@@ -154,9 +155,18 @@ def add_models_to_namespace(api_namespace):
     'password': fields.String(required=True, description='User\'s password')
 })
 
+# TODO: Remove 'expiry' after the android app refactoring.
 login_response_body_model = Model('Login response data model', {
     'access_token': fields.String(required=True, description='User\'s access token'),
-    'expiry': fields.Float(required=True, description='Access token expiry UNIX timestamp')
+    'expiry': fields.Float(required=True, description='Access token expiry UNIX timestamp'),
```

Comment:
@isabelcosta i kept it.

Reply:
Cool ;) it can be removed later @iulianav